### DISCUSSION TITLE
Can not calibration accelerometer in iNAV6.0

### DISCUSSION THREAD

**Discussion by tomhtm** [iNavFlight/inav#8548] [Q&A] [answered]
Hi all,
I can not calibration accelerometer when i load the default values "**Mini Quad with 3-5' propellers**" on the begaining.
The page of "Accelerometer calibration" display "Reset Accelerometer Calibration" with red font. and the FC will reboot when i click the Reset buttom.
If i don't load the default values, i can do the calibration accelerometer.

May i ask if not need do the "calibration accelerometer" when load the default values "**Mini Quad with 3-5' propellers**" ?

FC:HGLF722 mini
Firmware:inav_6.0.0_HGLRCF722
Configurator:6.0.0-FP2

**Comment by MrD-RC** [comment]
The **Reset Accelerometer Calibration** button should only be shown if there is a valid calibration already. If you have loaded a diff from a previous installation on this FC, the calibration will already exist. Clicking the **Reset Accelerometer Calibration** resets the accelerometer calibration, so that you can perform a fresh calibration.

**Comment by MrD-RC** [comment, marked as answer]
Anyone still using FP1 should update to RC1 (or later) firmware and configurator. 

An issue affecting some flight controllers was fixed in https://github.com/iNavFlight/inav-configurator/pull/1664
